src/components/WalletInit/CreateWallet/MnemonicCheckScreen.js

```diff
@@ -143,75 +142,79 @@ const MnemonicCheckScreen = ({
   const last = _.last(partialPhrase)
 
   return (
-    <SafeAreaView style={styles.safeAreaView}>
```

Comment:
What's the rationale for removing `SafeAreView`?

Reply:
my understanding is that `react-navigation` takes care of notches / status bars, etc...
so `<SafeAreaView>` would only be necessary in scenes that do not have a `react-navigation` header

Reply:
from the `react-navigation` docs
> React Navigation handles safe area in the default header. However, if you're using a custom header, it's important to ensure your UI is within the safe area.

Reply:
Actually they only handle it for their components (eg. the navigation bar). Quoting from their docs:

> While React Navigation handles safe areas for the built-in UI elements by default, your own content also needs to handle it to ensure that content isn't hidden by these items.
> 
> (...)
> 
> While React Native exports a SafeAreaView component, it has some inherent issues, i.e. if a screen containing safe area is animating, it causes jumpy behavior. In addition, this component only supports iOS 10+ with no support for older iOS versions or Android. We recommend to use the react-native-safe-area-context library to handle safe areas in a more reliable way.
> 

I don't have many devices to check if this doesn't have unexpected effects, and probably for these screens there shouldn't be any issue, but as a general rule I'd prefer that we always wrap the content inside a `SafeAreaView` from `react-native-safe-area-content` in particular 🙏

Reply:
my understanding is that `<SafeAreaView>` adds the top/bottom padding regardless if the component would overlap the notches.
for instance, this is what im seeing on the ios simulator
ive added red borders around the `<SafeAreaView>` and `<WalletForm>`
<img width="390" alt="Screen Shot 2021-04-26 at 13 14 30" src="https://user-images.githubusercontent.com/8462274/116144861-62069780-a691-11eb-82b5-cb3d94ea9c01.png">

Reply:
might be related
https://github.com/th3rdwave/react-native-safe-area-context/issues/107

Reply:
oh i see what's happening.

i agree now, that each of the main scene components should be wrapped in a <SafeAreaView>`, but if the navigator uses a `react-navigation` header or footer, then the `<SafeAreaView>` also needs to specify the `edges` prop  to avoid double padding 
(but only in ios for some weird reason)

ill take a look at this again tomorrow